addclear.js

```diff
@@ -64,15 +66,14 @@
 					me = this,
 					options = this.options;
 
-			$this.wrap("<span style='position:relative;' class='add-clear-span'></span>");
```

Comment:
Inline css is very annoying to override, so I removed it from this element so you can add it to your own css, or you can include a css file with the settings for the different elements, instead of putting it all in JS :) Thank you for the merge by the way, I'll pay more attention next time.